cmd/event_display/main.go

```diff
@@ -17,22 +17,19 @@ limitations under the License.
 package main
 
 import (
-	"bytes"
 	"context"
-	"encoding/json"
 	"fmt"
 	"log"
-	"strings"
 
-	"github.com/cloudevents/sdk-go/pkg/cloudevents"
-	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
+	"github.com/cloudevents/sdk-go"
 	"github.com/knative/eventing-sources/pkg/kncloudevents"
 )
 
 /*
 Example Output:
 
-☁  CloudEvent: valid ✅
+☁  cloudevents.Event:
+Validation: valid
```

Comment:
no emoji ??

Reply:
look at the rest of the diff... :(

the cloudevents spec was too serious for the emoji ...